### ISSUE TITLE
GG Galaxy

### ISSUE THREAD

**Issue by SulivanM** [JojoEffekt/GGStudio-SWF#1]
Hey do you have gg galaxy ?

**Comment by JojoEffekt** [comment]
no but i guess i could get the files if its absolutely necessary

**Comment by SulivanM** [comment]
Yes that can be cool for a revive project server like ggr reborn for cafe / disco / fashion so the galaxy is totally missing for files part.
If you can get it, tell me !

**Comment by JojoEffekt** [comment]
i will but i need some time
